clusterloader2/testing/access-tokens/config.yaml

```diff
@@ -0,0 +1,167 @@
+# Stress testing access token validation
+#
+# Targeting 2 000 tokens with 5 000 total QPS for 5k node cluster, so it's 2.5 QPS per token.
+#
+# Number of tokens does not scale with number of nodes.
+# By default, those 2 000 tokens are are assigned to 80 service accounts, with 25 tokens each.
```

Comment:
Sorry for nitpicking, but I'd also add a line about number of pods.
IIUC there will be always 80 pods, which means that this test can be run even on a 1 node cluster (assuming the node is big enough). If we had more pods then we'd run into 110pods/node limitation.